src/domain/checks/__tests__/data.ts

```diff
@@ -10,13 +10,13 @@ export const twoPreLoadRecordsOneUploadRecord = [data.preLoadRecord, data.preLoa
 export const evenPreLoadAndUploadRecordsThatAreLinked = [
   data.preLoadRecord,
   data.preLoadRecord,
-  data.uploadToS3Record,
-  data.uploadToS3Record,
+  { ...data.uploadToS3Record, loggedAt: new Date(703429620000) },
```

Comment:
What do you think to fill the correct date into `src/domain/__tests__/data.ts` based on the constant `aDate`? e.g.:

``` TypeScript
export const preLoadRecord: PreLoadRecord = {
  type: 'PreLoadRecord',
  input: { apiKey: apiKey.valid, body: [preLoadBody] },
  output: { statusCode: 200, returned: [preLoadResponse] },
  loggedAt: new Date(aDate.getTime() + 1000),
};
```

Reply:
Done! What do you think now?
I kept an upload record with the updated loggedAt field, just because I think it can be easier to understand that those uploadRecords are different.

Reply:
Changed my mind! 🃏